rpcs3/Emu/Io/MouseHandler.h

```diff
@@ -80,6 +80,10 @@ struct MouseData
 	s8 y_axis = 0;
 	s8 wheel = 0;
 	s8 tilt = 0; // (TODO)
+	s32 x_pos;
```

Comment:
The changes in this file make no sense.
This is cellMouse related code and is not supposed to be used in this way.